src/Native/build.proj

```diff
@@ -100,6 +102,9 @@
     <ItemGroup>
       <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MklImports$(NativeLibExtension)"
                           RelativePath="Microsoft.ML.HalLearners\runtimes\$(PackageRid)\native" /> 
+						  
+      <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MklImports$(NativeLibExtension)"
+                          RelativePath="Microsoft.ML\runtimes\$(PackageRid)\native" /> 						  
```

Comment:
We shouldn't be adding MklImports to the `Microsoft.ML` package. That was the reason we created `Microsoft.ML.HalLearners`, so every user of ML.NET doesn't need to download MklImports.